ts/screens/messages/MessagesHomeScreen.tsx

```diff
@@ -16,15 +17,23 @@ import { ScreenContentHeader } from "../../components/screens/ScreenContentHeade
 import TopScreenComponent from "../../components/screens/TopScreenComponent";
 import { MIN_CHARACTER_SEARCH_TEXT } from "../../components/search/SearchButton";
 import { SearchNoResultMessage } from "../../components/search/SearchNoResultMessage";
-import SectionStatusComponent from "../../components/SectionStatus";
+import SectionStatusComponent, {
+  InnerSectionStatus
+} from "../../components/SectionStatus";
 import FocusAwareStatusBar from "../../components/ui/FocusAwareStatusBar";
+import { unsupportedDeviceMoreInfoUrl } from "../../config";
```

Comment:
This URL is used only in this component, why don't we hardcode it in here and avoid placing it under the `.env` files and the `.config` one? It's unlikely for it to change or to be different between local and environment files and it will be removed in coming future versions of the application

Reply:
We prefer to leave this the way it is for now.